uc.c

```diff
@@ -611,17 +607,13 @@ uc_err uc_mem_read(uc_engine *uc, uint64_t address, void *_bytes, size_t size)
 
 UNICORN_EXPORT
 uc_err uc_mem_write(uc_engine *uc, uint64_t address, const void *_bytes,
```

Comment:
You forget to change the declaration in unicorn.h

Reply:
Thank you so much for pointing it out, because the `unicorn.h` is only used by program, and `size_t` is compatible with `uint64_t` on 64-bit platform, my compiler didn't complain about the inconsistency. My bad.